x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/precision_slider.tsx

```diff
@@ -0,0 +1,124 @@
+
+import { DOCS_PREFIX } from '../../../../routes';
+import { RelevanceTuningLogic } from '../../relevance_tuning_logic';
+
+import { STEP_DESCRIPTIONS } from './constants';
+
+export const PrecisionSlider: React.FC = () => {
+  const {
+    searchSettings: { precision },
+  } = useValues(RelevanceTuningLogic);
+
+  const { updatePrecision } = useActions(RelevanceTuningLogic);
+
+  const stepDescription = STEP_DESCRIPTIONS[precision];
+
+  return (
+    <>
+      <EuiTitle size="m">
+        <h2>
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.title',
+            {
+              defaultMessage: 'Precision tuning',
+            }
+          )}
+        </h2>
+      </EuiTitle>
+      <EuiSpacer />
+      <EuiPanel hasShadow={false} paddingSize="none">
+        <EuiText color="subdued">
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.description',
+            {
+              defaultMessage: 'Fine tune the precision vs. recall settings on your engine.',
+            }
+          )}{' '}
+          <EuiLink
+            data-test-subj="documentationLink"
+            href={`${DOCS_PREFIX}/precision-tuning.html`}
+            target="_blank"
+          >
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.learnMore.link',
+              {
+                defaultMessage: 'Learn more.',
+              }
+            )}
+          </EuiLink>
+        </EuiText>
+        <EuiSpacer />
+        <EuiFlexGroup justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <strong>
+              {i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.recall.label',
+                {
+                  defaultMessage: 'Recall',
+                }
+              )}
+            </strong>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <strong>
+              {i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.precision.label',
+                {
+                  defaultMessage: 'Precision',
+                }
+              )}
+            </strong>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+        <EuiSpacer size="s" />
+        <EuiRange
+          data-test-subj="PrecisionRange"
+          value={precision}
+          onChange={(e) => {
+            updatePrecision(parseInt((e.target as HTMLInputElement).value, 10));
+          }}
+          min={1}
+          max={11}
+          step={1}
+          showRange
+          showTicks
+          fullWidth
+        />
+        {stepDescription && (
+          <>
+            <EuiSpacer size="s" />
+            <EuiPanel
+              color="subdued"
+              data-test-subj="StepDescription"
+              style={{ minHeight: '4.7143rem' }}
```

Comment:
Where did this number come from? It feels really oddly specific 😬 Can we round it off?

Can we prefer a CSS class here so we can use `$euiSize`? Maybe try this, it looks like there's no visual bouncing issues with a x4:


```scss
.precisionTuningTooltip {
  min-height: $euiSize * 4;
}
```